bin/io.js

```diff
@@ -0,0 +1,56 @@
+// this module just contains stuff used for io
```

Comment:
I don't think a comment like this adds value.  This file should have a copyright notice.

Reply:
Removed it in https://github.com/digitalbazaar/http-signature-header/pull/12/commits/bec870709f6558367a7882308bb1b81094cc673c